render_pass.py

```diff
@@ -76,6 +87,42 @@ def step_callback(step, width=None, height=None, shared_memory_name=None):
                             
                             combined_pixels = np.empty((size_x * size_y, 4), dtype=np.float32)
                             rect.foreach_get(combined_pixels)
+
+                            event = threading.Event()
+                            def do_ocio_transform(event):
+                                ocio_config_path = os.path.join(bpy.utils.resource_path('LOCAL'), 'datafiles/colormanagement/config.ocio')
+                                buf = combined_pixels.tobytes()
+                                combined_pixels_memory = SharedMemory(create=True, size=len(buf))
+                                combined_pixels_memory.buf[:] = buf
+                                args = {
+                                    'config_path': ocio_config_path,
+                                    'name': combined_pixels_memory.name,
+
+                                    'exposure': scene.view_settings.exposure,
+                                    'gamma': scene.view_settings.gamma,
+                                    'view_transform': scene.view_settings.view_transform,
+                                    'display_device': scene.display_settings.display_device,
+                                    'look': scene.view_settings.look,
+                                }
+                                def image_callback(event, shared_memory_name, seed, width, height, upscaled=False):
+                                    nonlocal combined_pixels
+                                    shared_memory = SharedMemory(shared_memory_name)
+                                    combined_pixels = np.frombuffer(shared_memory.buf, dtype=np.float32).copy().reshape((size_x * size_y, 4))
+                                    shared_memory.close()
+                                    event.set()
+                                def exception_callback(fatal, msg, trace):
+                                    print(fatal, msg, trace)
+                                    event.set()
+                                generator_advance = GeneratorProcess.shared().apply_ocio_transforms(args, functools.partial(image_callback, event), exception_callback)
+                                while True:
+                                    try:
+                                        next(generator_advance)
+                                    except StopIteration:
+                                        break
+
+                            bpy.app.timers.register(functools.partial(do_ocio_transform, event))
+                            event.wait()
```

Comment:
As in, apply an inverse transformation on the final output from dream textures so the color management options give you the generated image back unmodified?

Reply:
Yes, that way when Blender does the final color transform it'll hopefully be colored the same as before the inverse transform and not become overexposed.

Reply:
I applied an inverse, let me know if this has the intended effect.

Reply:
Now it's behaving like no color transformation was applied, so I guess an inverse wasn't quite what it needed. Maybe instead of inverse it can transform specifically to `View Transform: Raw, Look: None, Exposure: 0, Gamma: 1`?

Reply:
Though as it currently is, adding a convert colorspace node on the dream textures pass converting from linear to xyz seems to be very close to the colors it should be, and I think the difference at this point is stable diffusion img2img colorshifting.

Reply:
Ok, I will just leave it as is. I believe doing an inverse transform is the closest it can get to the intended output.